tests/lib/rules/valid-v-slot.js

```diff
@@ -289,8 +313,38 @@ tester.run('valid-v-slot', rule, {
           </MyComponent>
         </template>
       `,
+      options: [{ allowModifiers: true }],
       errors: [{ messageId: 'disallowAnyModifier' }]
     },
+    {
+      code: `
+        <template>
+          <MyComponent>
+            <template v-slot:foo.bar></template>
+          </MyComponent>
+        </template>
+      `,
+      options: [{ allowModifiers: false }],
+      errors: [{ messageId: 'disallowAnyModifier' }]
+    },
+    {
+      code: `
+        <template>
+          <MyComponent>
+            <template v-slot:foo></template>
+            <template v-slot:foo.bar></template>
+            <template v-slot:foo.baz></template>
+            <template v-slot:foo.bar.baz></template>
+          </MyComponent>
+        </template>
+      `,
+      options: [{ allowModifiers: false }],
+      errors: [
+        { messageId: 'disallowAnyModifier' },
+        { messageId: 'disallowAnyModifier' },
+        { messageId: 'disallowAnyModifier' }
+      ]
+    },
```

Comment:
Could you add test cases to check that the following code will result in errors with `"allowModifiers: true"`?

```vue
<template>
<MyComponent>
<template v-slot:foo.bar></template>
<template v-slot:foo.bar></template>
</MyComponent>
</template>
```